docs/administration/segregation.md

```diff
@@ -22,63 +18,62 @@ In OpenCTI, data is segregated based on knowledge marking. The diagram provided
 
 ![Data access diagram](assets/data-access-diagram.png)
 
-### Traffic Light Protocol
-
-The Traffic Light Protocol is implemented by default as marking definitions in OpenCTI. It allows you to segregate information by TLP levels in your platform and restrict access to marked data if users are not authorized to see the corresponding marking.
 
-The *Traffic Light Protocol* (TLP) was designed by the *Forum of Incident Response and Security Teams* ([FIRST](https://www.first.org/tlp/)) to provide a standardized method for classifying and handling sensitive information, based on four categories of sensitivity.
+## Manage markings
 
-For more details, the diagram provided below illustrates how are categorized the marking definitions:
+### Create new markings
 
-![TLP diagram](assets/tlp-diagram.png)
+To create a marking, you must first possess the capability `Manage marking definitions`. For further information on user administration, please refer to the [Users and Role Based Access Control page](./users.md).
 
-### Create new markings
+Once you have access to the settings, navigate to "Settings > Security > Marking Definitions" to create a new marking.
 
-In order to create a marking, you must first have the ability to access the Settings tab. For example, a user who is in a group with the role of Administrator can bypass all capabilities or a user who is in a group with the role that has `Access administration` checked can access the Settings tab. For more details about user administration, please see the [Users and Role Based Access Control page](./users.md).
+A marking consists of the following attributes:
 
-![Authorize access administration](assets/authorize-access.gif)
```

Comment:
I really like the gifs for showing the steps of a process!!!

Reply:
It's not from me. But yes, the gifs in the docs are great!